aviary/interface/methods_for_level2.py

```diff
@@ -485,39 +486,39 @@ def check_and_preprocess_inputs(self):
                             f"Invalid target_distance in [{phase_name}].[user_options]. "
                             f"Current (value: {target_distance[0]}), (units: {target_distance[1]}) <= 0")
 
-        # target_duration verification for all phases
         # Checks to make sure target_duration is positive,
-        # duration_bounds and initial_guesses for time have not been set,
-        # Then sets duration_bounds, initial_guesses, and fixed_duration
+        # Sets duration_bounds, initial_guesses, and fixed_duration
         for idx, phase_name in enumerate(self.phase_info):
             if 'user_options' in self.phase_info[phase_name]:
+                analytic = False
+                if (self.analysis_scheme is AnalysisScheme.COLLOCATION) and (self.mission_method is EquationsOfMotion.TWO_DEGREES_OF_FREEDOM):
+                    try:
+                        # if the user provided an option, use it
+                        analytic = self.phase_info[phase_name]["user_options"]['analytic']
+                    except KeyError:
+                        # if it isn't specified, only the default 2DOF cruise for collocation is analytic
+                        if 'cruise' in phase_name:
+                            analytic = self.phase_info[phase_name]["user_options"]['analytic'] = True
+                        else:
+                            analytic = self.phase_info[phase_name]["user_options"]['analytic'] = False
+
                 if 'target_duration' in self.phase_info[phase_name]["user_options"]:
                     target_duration = self.phase_info[phase_name]["user_options"]["target_duration"]
                     if target_duration[0] <= 0:
                         raise ValueError(
                             f"Invalid target_duration in phase_info[{phase_name}][user_options]. "
                             f"Current (value: {target_duration[0]}), (units: {target_duration[1]}) <= 0")
-                    if 'duration_bounds' in self.phase_info[phase_name]["user_options"]:
```

Comment:
Are we losing these 3 checks, or did they move elsewhere?

Reply:
@ehariton we removed these because the duration bounds just get updated based on the target_duration, right?

Reply:
yeah so there is some explained magic under the hood now that replaces a few elements based on `target_duration`. I detail that in the docs:

When creating a `"target_duration"` reserve phase there are a number of values inside of `phase_info['user_options']` that are overwritten in `check_and_preprocess_inputs()`. 
Specifically, `duration_bounds`, `fixed_duration`, and `"initial_guesses": {"times"}` will be over-written.